src/lib/keys.js

```diff
@@ -222,3 +222,30 @@ export const segmentNetworkKey = hex => {
 
   return [rowFrom(2), rowFrom(18), rowFrom(34), rowFrom(50)];
 };
+
+// ctsy @yosoyubik
+/**
+ * @param {object} pair
+ * @param {number} point
+ * @param {number} revision
+ * @return {string}
+ */
+export const compileMultikey = (point, pair1, pair2, revision1, revision2) => {
```

Comment:
@Fang- Please pay special attention to this!

Reply:
`[[1 0] ship [life1 key1] [life2 key2] 0]`

Reply:
Maybe I should've re-communicate as things drew closer, but I had already done multikeyfile work in 9331dd1999.

This seems to behave identically, but my version supports any number of keys in the file. (Additionally, small stylistic nit, `noun.Atom.yes` is semantically misleading, `noun.Atom.fromInt(0)` is nicer. Really there should just be a `noun.Atom.null` alias.)